client/src/inside/components/mar/MedList.vue

```diff
@@ -1,26 +1,19 @@
 <template lang="pug">
   ul
     li(class="medList", v-for="med in medsList")
-      h6 {{ med.medication }}
-      div {{ med.route }}
-      div {{ med.reason }}
+      div {{ med.medication }} {{ med.route }} {{ med.reason }}
 </template>
 
 <script>
-import MedOrder from './med-entity'
 export default {
-  name: 'MedList',
   props: {
     medsList: { type: Array }
   },
-  methods: {
-    medText (med) {
-      let text =  MedOrder.medOrderAsTextLine(med)
-      // console.log('MedList medText', text)
-      return text
-    }
+  mounted: function () {
+    console.log('this.medLsit >> ', this.medsList)
```

Comment:
comment this out or put it behind a if(debug) flag so that it is not visible unless a developer enables it